catalog/core/catalog-core-classification-api/src/main/java/org/codice/alliance/catalog/core/internal/api/classification/SecurityClassificationService.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.catalog.core.internal.api.classification;
+
+import java.util.Comparator;
+
+/**
+ * <b> This code is experimental. While this interface is functional and tested, it may change or be
+ * removed in a future version of the library. </b>
+ */
+public interface SecurityClassificationService {
+
+    /**
+     * Get a comparator for comparing two security classification strings. Classification strings
+     * will be sorted so that the lowest security is first and the highest security is last.
+     *
+     * @return non-null value
+     */
+    Comparator<String> getSecurityClassificationComparator();
```

Comment:
I think the service could extend Comparator. To me, it felt strange having a comparator also be a service.